From 3ad2ac58e892ac131d18f4b33fabc9d57f9d97dd Mon Sep 17 00:00:00 2001
From: "Frederic Pierret (Epitre)" <frederic.epitre@orange.fr>
Date: Tue, 19 Sep 2017 18:36:26 +0200
Subject: [PATCH 19/29] anaconda: skip NTP installation and setup in dom0

Dom0 has no direct network access, to this doesn't make sense anyway.

Fixes QubesOS/qubes-issues#2110
---
 pyanaconda/kickstart.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/kickstart.py b/pyanaconda/kickstart.py
index 8639754..d65e44e 100644
--- a/pyanaconda/kickstart.py
+++ b/pyanaconda/kickstart.py
@@ -1744,6 +1744,9 @@ class Timezone(commands.timezone.F25_Timezone):
         self._disabled_chrony = False
 
     def setup(self, ksdata):
+        ### Skip the whole NTP setup in Qubes dom0
+        return
+
         # do not install and use NTP package
         if self.nontp or NTP_PACKAGE in ksdata.packages.excludedList:
             if iutil.service_running(NTP_SERVICE) and \
-- 
2.9.5

